src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties

```diff
@@ -378,6 +378,18 @@ doclet.help.search.refer=Refer to the {0} for a full description of search featu
 doclet.help.search.spec.url=https://docs.oracle.com/en/java/javase/{0}/docs/specs/javadoc/javadoc-search-spec.html
 # The title for the Javadoc Search Specification
 doclet.help.search.spec.title=Javadoc Search Specification
+doclet.help.releases.head=Release Details
```

Comment:
The subtitle "Release Details" surprised me, because it seems to suggest "details about releases", while it is more like "release information (contained in API details)".

Reply:
Sorry, revisiting this now I think "Release Details" is absolutely fine. I must have been overly nit-picky that day.